src/client/rewards/CampaignFooter/CampaignButtons.js

```diff
@@ -210,17 +198,12 @@ export default class CampaignButtons extends React.Component {
           {this.renderPostPopoverMenu()}
         </div>
         <React.Fragment>
-          <LinkButton
-            block={false}
-            className="WriteReview-button"
-            to={`/editor?object=[${requiredObjectName}](${requiredObjectPermlink})&object=[${proposedObjectName}](${proposedObjectPermlink})&campaign=${propositionId}`}
-            type="primary"
-          >
+          <Button type="primary" onClick={() => toggleModalDetails({ value: true })}>
```

Comment:
toggleModalDetails({ value: true }) - does it open modal? Better to add openModalDetails method to current component

Reply:
++